app/views/job_runs/_structural_updates_summary.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="structural-changes">
+    <table class="table">
+    <caption>Updates from csv</caption>
+      <thead class="table">
+        <tr>
+          <th>Druid</th>
+          <th>Filename</th>
+          <th>Status</th>
+        </tr>
+      </thead>
+
+    <% druids.each do |druid| %>
+      <tbody>
```

Comment:
Do you want one `<tbody>` for each druid, or just one `<tbody>` for the whole table? I might have expected the latter.

Reply:
Whoops, good catch, just put that in the wrong place.